CMS-Enterprise/easi-app
# [EASI-4488] GRB Review supporting docs table

# EASI-4488

## Description

- Adds the supporting documents table to the GRB Review page ([Figma](https://www.figma.com/design/G0UN35V7SY0g9nkTIESxsY/EASi-GRB-Experience?node-id=2597-48335&t=1zMmvxZaJFT2DmCu-0)).
- Note: Navigation link work was separated out into [4583](https://jiraent.cms.gov/browse/EASI-4583).

## How to test this change

- Start the app 
- Run the seed scripts with `scripts/dev db:clean && scripts/dev db:seed`
- Clean the documents so you can see the view/delete buttons with `scripts/dev minio:clean`
- [Navigate to this intake GRB Review page with multiple reviewers](http://localhost:3000/governance-review-team/5af245bc-fc54-4677-bab1-1b3e798bb43c/grb-review).
    - USR1 is the requester for this intake, but you may sign in with `A11Y` who is assigned as a reviewer to verify you cannot delete documents. You can also add documents as an admin that is not the requester with this job code for additional testing.

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
